charts/vela-core/values.yaml

```
@@ -88,19 +88,23 @@ healthCheck:
 ##@param optimize.markWithProb Optimize ResourceTracker GC by only run mark with probability. Side effect: outdated ResourceTracker might not be able to be removed immediately.
 ##@param optimize.disableComponentRevision Optimize componentRevision by disabling the creation and gc
 ##@param optimize.disableApplicationRevision Optimize ApplicationRevision by disabling the creation and gc.
-##@param optimize.disableWorkflowRecorder Optimize workflow recorder by disabling the creation and gc.
 ##@param optimize.enableInMemoryWorkflowContext Optimize workflow by use in-memory context.
 ##@param optimize.disableResourceApplyDoubleCheck Optimize workflow by ignoring resource double check after apply.
 ##@param optimize.enableResourceTrackerDeleteOnlyTrigger Optimize resourcetracker by only trigger reconcile when resourcetracker is deleted.
+##@param optimize.informerCache if enabled, informer cache will be optimized by removing unnecessary fields will be filtered
+##@param optimize.applicationRevisionDefinitionStorage if enabled, application revision definition cache will be optimized by using reducing duplicated definition storage
+##@param optimize.disableWorkflowContextConfigMapCache if enabled, disable the workflow context's configmap informer cache
 optimize:
   cachedGvks: ""
   markWithProb: 0.1
   disableComponentRevision: true
   disableApplicationRevision: false
-  disableWorkflowRecorder: false
   enableInMemoryWorkflowContext: false
   disableResourceApplyDoubleCheck: false
   enableResourceTrackerDeleteOnlyTrigger: true
+  informerCache: true
+  applicationRevisionDefinitionStorage: true
+  disableWorkflowContextConfigMapCache: true
```

Shall we provide a new optimize section for these fine-grained cache items?


```suggestion
  disableCacheItems:
    disableWorkflowContextConfigMapCache: false
```

BTW, the parameter name is confusion, it's default to disable the cache or not?

---

I think it should be 
```suggestion
    disableCacheItems:
	    workflowContextConfigMapCache: false
```

---

Changed to default enabled featuregate